src/scss/components/paragraphs/_chart.scss

```diff
@@ -0,0 +1,25 @@
+$_chart-aspect-ratio: 1.78; // 16/9
+
+.chart {
+  figure {
+    // Reset user-agent stylesheet rules for figure.
+    margin: 0;
```

Comment:
I'm wondering if we should consider resetting these globally in our reset for all figures?